implementations/rust/kex/src/x3dh.rs

```diff
@@ -413,9 +414,9 @@ impl KeyExchanger for X3dhInitiator {
 
                 // Check the prekey_bundle signature
                 vault.verify(
-                    prekey_bundle.signature_prekey.0,
-                    prekey_bundle.identity_key.0.as_slice(),
-                    prekey_bundle.signed_prekey.0.as_slice(),
+                    prekey_bundle.signature_prekey.as_ref().clone(), /* FIXME */
```

Comment:
@mikelodder7 should verify take array reference instead of array?

Reply:
yes it should

Reply:
@mikelodder7 addressed in #704